### ISSUE TITLE
Fix the navbar toggle button issue

### ISSUE THREAD

**Issue by ayushnagar123** [codeuino/codebadge-frontend#3]
#### Is your feature request related to a problem? Please describe.
The navbar toggle is not working properly in mobile view need to be fixed.

#### Describe the solution you'd like
Use flexbox and bootstrap to fix the issue

Additional context
* navbar before the toggle
![image](https://user-images.githubusercontent.com/32647423/83466776-ccb94e00-a495-11ea-84ab-29b28a7b95ff.png)
* navbar when open
![image](https://user-images.githubusercontent.com/32647423/83466745-b3180680-a495-11ea-969a-e1fbc46ad168.png)

**Comment by jaskiratsingh2000** [comment]
Interesting! Responsiveness is an issue

**Comment by ayushnagar123** [comment]
Yes. we can fix that not a big issue. Will discuss this with Siddharth regarding dimensionality fits of these components. Just Made the design scalable for the present scenario. Designs are fine for all views above 768px for any desktop width at present. Rest will fix some minor bugs in discussion with him.
